flake.nix

```diff
@@ -4,19 +4,21 @@
     flake-utils.url = "github:numtide/flake-utils";
     nix-filter.url = "github:numtide/nix-filter";
 
-    ocaml-overlay.url = "github:anmonteiro/nix-overlays";
-    ocaml-overlay.inputs.nixpkgs.follows = "nixpkgs";
-    ocaml-overlay.inputs.flake-utils.follows = "flake-utils";
+    ocaml-overlays.url = "github:anmonteiro/nix-overlays";
```

Comment:
Renaming from `overlay` to `overlays` and use the same syntax as Deku to be conssitent.